File: minegame-server/src/main/java/top/nipuru/minegame/server/game/chat/ChatData.java
package top.nipuru.minegame.server.game.chat;

import lombok.Getter;
import lombok.experimental.Accessors;
import top.nipuru.minegame.server.game.player.Data;
import top.nipuru.minegame.server.game.player.Table;
import top.nipuru.minegame.server.game.player.Temp;

@Getter
@Accessors(fluent = true)
@Table(name = "tb_chat")
public class ChatData implements Data {

    /** 禁言截止时间 */
    long mute;

    /** 私聊玩家名 */
    @Temp String msgTarget = EMPTY_STRING;
}

File: minegame-game/src/main/kotlin/top/nipuru/minegame/game/gameplay/chat/ChatData.kt
package top.nipuru.minegame.game.gameplay.chat

import top.nipuru.minegame.game.gameplay.player.Data
import top.nipuru.minegame.game.gameplay.player.Table
import top.nipuru.minegame.game.gameplay.player.Temp

@Table(name = "tb_chat")
class ChatData : Data {
    /** 禁言截止时间  */
    var mute: Long = 0

    /** 私聊玩家名  */
    @Temp
    var msgTarget: String = ""
}
